boards/others/stm32_min_dev/board.yml

```diff
@@ -4,5 +4,9 @@ board:
   vendor: others
   revision:
     format: custom
+    default: "blue"
```

Comment:
This is wrong, this should never have been made the default, these are completely different boards by completely different vendors that have no relation to one above and there should be no default

Reply:
Can you submit a follow-up PR showing how you mean it should have been done?

Reply:
It should be reverted, it should just not have a default, you build for stm32_min_dev@blue or stm32_min_dev@black, specifying the revision

Reply:
I see there is a default in the revisions file from the hwmv2 conversion, that should not have been added